src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/schemas/EmptySchema.kt

```diff
@@ -0,0 +1,32 @@
+package org.vaccineimpact.api.blackboxTests.schemas
+
+import org.assertj.core.api.Assertions.assertThat
+import org.vaccineimpact.api.validateSchema.Validator
+
+object EmptySchema : Schema
+{
+    override val validator: Validator = EmptyValidator()
+
+    override fun validateResponse(response: String, contentType: String?)
+    {
+        assertThat(response).isBlank()
+    }
+
+    override fun validateRequest(request: String)
+    {
+        assertThat(request).isBlank()
+    }
+
+    class EmptyValidator : Validator
```

Comment:
This is never used right? Why not just set the `validator` to be an instance of the existing `JsonValidator`? Or throw an exception on invocation like we do with the `CSVSchema` (as presumably we don't ever want a blank response?)

Reply:
Unless I'm missing something!

Reply:
Yeah, you're right - I was just blindly implementing the interface. Fixed.